configs/components/jsbach/jsbach.datasets.yaml

```
@@ -2,6 +2,7 @@ input_sources:
         lctlib: "${echam.model_dir}/lctlib_nlct21.def"
         "jsbach_1850": "${adj_input_dir}/jsbach_${echam.resolution}${echam.ocean_resolution}_11tiles_5layers_1850${jsbachnc_file_ending}.nc"
         "jsbach_1976": "${adj_input_dir}/jsbach_${echam.resolution}${echam.ocean_resolution}_11tiles_5layers_1976${jsbachnc_file_ending}.nc"
+        "jsbach_ismc": "${general.experiment_couple_dir}/latest_jsbach_init_file.nc"
```

Can you put a comment here what ismc means? I suppose ice sheet model coupled, but I will probably look at this in a year or two and completely forget what it is.

```suggestion
        # Input for JSBACH with Ice Sheet Model Coupling Enabled
        "jsbach_ismc": "${general.experiment_couple_dir}/latest_jsbach_init_file.nc"
```